web/modules/custom/ingredient_validation/src/Plugin/Validation/Constraint/IngredientConstraint.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\ingredient_validation\Plugin\Validation\Constraint;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * Make sure that the ingredients are written correctly.
+ *
+ * @Constraint(
+ * id = "ValidIngredient",
+ * label = @Translation("Prevent Product CT creation
+ * if it isn't matched with the required formula", context = "Validation"),
```

Comment:
This should be a single line commit. a short human readable label for the validator.